src/commands/kat-tests-install-tools.yaml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  kind-version:
```

Comment:
How are these exposed? Can I change them through these parameters? I can't see that in the example.

Reply:
These are not user-facing parameters (at least yet), rather templating vars to make version changes easier in the future (and potentially configurable - not really supported right now).

Reply:
Okay - we should somehow make sure that we remember these / document these. Otherwise I see us diverging in versions here which is not something we want.

Reply:
Do we have a "source fo truth" for versions we should use currently? Is it possible to get it automatically?

Reply:
Not that I am aware of right now - something to consider for @giantswarm/wg-integration-testing  as we will use kind and similar tools there as well.

Reply:
Yes having this aligned between the integration test job and here would be nice. There the kind version is hardcoded but the kubernetes version is a param.

I'm fine with this approach but yes I think something for the WG. Maybe a standard location for all the versions (kind, kubernetes, helm) would help and avoid them drifting. I'll put a note in the board.

Reply:
Yeap, taht would be cool - to have a source of versions that should be used. Please ping me in the new issue as well, I would like to integrate this later with this tool.